app/shaders/despill.frag

```diff
@@ -0,0 +1,51 @@
+uniform sampler2D tex_in;
+uniform int color_in;
+uniform int method_in;
+uniform bool preserve_luminance_input;
+uniform vec3 luma_coeffs;
+
+varying vec2 ove_texcoord;
+
+void main(void) {
+    vec4 tex_col = texture2D(tex_in, ove_texcoord);
+
```

Comment:
Fortunately, GLSL allows the use of switch cases and converting multiply-add component math to dot products. I'll make some changes and optimizations